JLJSONMapping/Category/NSError+JLJSONMapping.h

```diff
@@ -8,10 +8,6 @@
 
 #import <Foundation/Foundation.h>
 
-FOUNDATION_EXPORT NSString * const kFABObjectMappingDomain;
```

Comment:
These were causing import issues while I had been making changes to the Framework loading in the sample app.